mlflow/models/evaluation/base.py

```
@@ -909,6 +956,12 @@ def scatter_plot(eval_df, builtin_metrics, artifacts_dir):
             "The model argument must be a string URI referring to an MLflow model or "
             "an instance of `mlflow.pyfunc.PyFuncModel`."
         )
+    if isinstance(baseline_model, str):
+        baseline_model = mlflow.pyfunc.load_model(baseline_model)
+    elif baseline_model is not None:
+        raise ValueError(
```

Can we raise `MlflowException` with error code `INVALID_PARAMETER_VALUE`?